indico/modules/rb/client/js/common/config/reducers.js

```diff
@@ -17,20 +17,29 @@ export const initialState = {
     languages: {},
     tileServerURL: '',
     gracePeriod: 1,
+    helpURL: '',
+    tosURL: null,
+    tosHtml: null,
+    contactEmail: null
 };
 
 export default combineReducers({
     request: requestReducer(configActions.FETCH_REQUEST, configActions.FETCH_SUCCESS, configActions.FETCH_ERROR),
     data: (state = initialState, action) => {
         switch (action.type) {
             case configActions.CONFIG_RECEIVED: {
-                const {roomsSpriteToken, tileserverURL: tileServerURL, gracePeriod} = camelizeKeys(action.data);
+                const {roomsSpriteToken, tileserverURL: tileServerURL, gracePeriod, helpURL, contactEmail, tosURL,
+                       tosHtml} = camelizeKeys(action.data);
```

Comment:
`tosHtml: tosHTML` so you don't have to use the awful mixed-cased acronym spelling ;)